<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="mangesh-Inspiron-354" failures="0" tests="4" name="listerner.SampleTest1" time="0.531" errors="1" timestamp="2023-06-29T19:10:25 IST" skipped="1">
  <testcase classname="listerner.SampleTest1" name="TC4" time="0.003">
    <system-out>
      <![CDATA[ThisTC4execution started
TC4 is running
ThisTC4 method completed successfully]]>
    </system-out>
  </testcase> <!-- TC4 -->
  <testcase classname="listerner.SampleTest1" name="TC1" time="0.003">
    <system-out>
      <![CDATA[ThisTC1execution started
TC1 is running
ThisTC1 method completed successfully]]>
    </system-out>
  </testcase> <!-- TC1 -->
  <testcase classname="listerner.SampleTest1" name="TC2" time="0.525">
    <error message="Method listerner.SampleTest1.TC2() didn&#039;t finish within the time-out 500" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method listerner.SampleTest1.TC2() didn't finish within the time-out 500
at java.base@17.0.5/java.lang.Thread.sleep(Native Method)
at app//listerner.SampleTest1.TC2(SampleTest1.java:19)
at java.base@17.0.5/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.5/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.5/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.5/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.5/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.5/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.5/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.5/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
    <system-out>
      <![CDATA[ThisTC2execution started
ThisTC2method is Failed,pl try again]]>
    </system-out>
  </testcase> <!-- TC2 -->
  <testcase classname="listerner.SampleTest1" name="TC3" time="0.000">
    <skipped/>
    <system-out>
      <![CDATA[This TCTC3is skipped..plz check dependent method]]>
    </system-out>
  </testcase> <!-- TC3 -->
  <system-out>
    <![CDATA[ThisTC2execution started
ThisTC2method is Failed,pl try again
ThisTC1execution started
TC1 is running
ThisTC1 method completed successfully
ThisTC4execution started
TC4 is running
ThisTC4 method completed successfully
ThisTC3execution started
This TCTC3is skipped..plz check dependent method]]>
  </system-out>
</testsuite> <!-- listerner.SampleTest1 -->
